app/controllers/prices_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PricesController < ApplicationController
```

Comment:
I think ProductsController makes more sense to me, since we search for products. @hiattp what do you think?